### PR TITLE
fix pv_model documentation

### LINKED ISSUE #684: Fix documentation of PV Model
I've noticed some flaws in the PV Model documentation that should be fixed. These are:

-Add description of Φ in the calculation of Sunrise Angle
-Swap out Sunrise- and Sunset Angle (sunrise angle should be negative, also matching the code)
-Fix gamma_e in Beam Radiation on Sloped Surface calculation
-Fix Incidence Angle: some terms are printed in italics
-Add description of E_dir,S in Beam Radiation on Sloped Surface calculation
-Remove description of alpha_s in Incidence Angle calculation
-Fix description of alpha_e: Should be "surface azimutz angle" instead of "sun azimuth"  (also matching the code)

### PR DESCRIPTION
**Pull request by SimonHuette**
resolves #684

### CODE REVIEW

**Comment by SimonHuette** [comment]
I have also noticed that certain formulas are written partially in italics, others not (inconsistent). In my opinion it looks better without italics, however let me know if this was done on purpose.

File: docs/readthedocs/models/pv_model.md [hunk 1]
```diff
@@ -30,7 +30,7 @@ To calculate the overall feed in of the pv unit, the sum of the direct radiation
 
 **Caution:** all angles are given in radian!
 
-The azimuth angle $\alpha_{E}$ starts at negative values in the East and moves over 0° (South) towards positive values in the West. [Source](https://www.photovoltaik.org/wissen/azimutwinkel)
+The surface azimuth angle $\alpha_{E}$ starts at negative values in the East and moves over 0° (South) towards positive values in the West. [Source](https://www.photovoltaik.org/wissen/azimutwinkel)
```
Comment (danielfeismann) [inline docs/readthedocs/models/pv_model.md:33]:
```suggestion
The surface azimuth angle $\alpha_{E}$ starts at negative values in the East and moves over 0° (South) towards positive values in the West. [(Source)](https://www.photovoltaik.org/wissen/azimutwinkel)
```

File: docs/readthedocs/models/pv_model.md [hunk 2]
```diff
@@ -112,18 +112,19 @@ $$
...
 $$
 
 $$
-\omega_{SS}=-\omega_{SR}
+\omega_{SR}=-\omega_{SS}
 $$
 
 *with*\
 **$\delta$** = the declination angle
+**$\phi$** = observer's latitude
```
Comment (danielfeismann) [inline docs/readthedocs/models/pv_model.md:127]: there should be a new line for each element

File: docs/readthedocs/models/pv_model.md [hunk 3]
```diff
@@ -202,11 +202,11 @@ $$
 Calculating the air mass ratio by dividing the radius of the earth with approx. effective height of the atmosphere (each in kilometer)
 
 $$
-airmassratio = (\frac{6371 km}{9 km}) = 707.8\overline{8}
+\airmassratio = (\frac{6371 km}{9 km}) = 707.8\overline{8}
 $$
 
 $$
-airmass = \sqrt{(707.8\overline{8} \cdot \cos({\theta_z}))^2 +2 \cdot 707.8\overline{8} +1)} - 707.8\overline{8} \cdot \cos{(\theta_z)})
+\airmass = \sqrt{(707.8\overline{8} \cdot \cos({\theta_z}))^2 +2 \cdot 707.8\overline{8} +1)} - 707.8\overline{8} \cdot \cos{(\theta_z)})
```
Comment (danielfeismann) [inline docs/readthedocs/models/pv_model.md:209]: This is not rendered well

File: docs/readthedocs/models/pv_model.md [hunk 4]
```diff
@@ -264,10 +264,10 @@ From here on, formulas from given reference below are used:
 
 $$
 \begin{eqnarray*}
-a = (\sin(\delta) \cdot \sin(\phi) \cdot \cos(\gamma_{e})
+a = (\sin(\delta) \cdot \sin(\phi) \cdot \cos(\gamma_{e}) -
```
Comment (danielfeismann) [inline docs/readthedocs/models/pv_model.md:267]: Good catch!

**Review by danielfeismann — CHANGES_REQUESTED** [review]
Thanks for updating the documentation. Good catches! Just some minor remarks from my side

File: docs/readthedocs/models/pv_model.md (hunk 3 above)
Comment (SimonHuette) [reply docs/readthedocs/models/pv_model.md:209]: Can you specify what exactly the problem is?

Comment (danielfeismann) [reply docs/readthedocs/models/pv_model.md:209]: Sure, sorry. Next time I should directly add a screenshot. 

grafik

see generated docu here: [https://simona--685.org.readthedocs.build/en/685/models/pv_model.html](https://simona--685.org.readthedocs.build/en/685/models/pv_model.html)

**Review by danielfeismann — APPROVED** [review]
Thanks for improving the docs.

**Pull request merged**